.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

github action 에러 코드를 보니까 npm ci에서 에러가 나더라구요?? 그래서 찾아보니까 node-version을 14로 바꿔줘야한다는 것 같습니당!!
```suggestion
        node-version: [14.x]
```
이렇게 한번 바꿔주시죵!

---

덕분에 검색을 해보니,
node12는 22년 4월부터 지원이 중단되어서 노드 12 대신 16으로 업데이트를 하라는 부분들이 있던데
최신 버전은 불안정한 부분들이 있어서 14버전으로 바꾸어주는 걸까요?